openmdao/drivers/doe_driver.py

```diff
@@ -96,11 +99,8 @@ def _declare_options(self):
         self.options.declare('generator', types=(DOEGenerator), default=DOEGenerator(),
                              desc='The case generator. If default, no cases are generated.')
 
-        self.options.declare('parallel', default=False, types=(bool, int), lower=0,
-                             desc='True or number of cases to run in parallel. '
-                                  'If True, cases will be run on all available processors. '
-                                  'If an integer, each case will get COMM.size/<number> '
-                                  'processors and <number> of cases will be run in parallel')
+        self.options.declare('procs_per_model', default=1, lower=1,
+                             desc='Number of processors to give each model under MPI.')
```

Comment:
feature doc needs to be updated for this as well

Reply:
fixed the typos

Reply:
Working on this, pending an API question for jubs.